src/components/WorkSection.js

```diff
@@ -128,15 +129,28 @@ class WorkSection extends PureComponent {
                     <h2 className="paragraph mb_5">
                       {get(work, 'fields.title', '')}
                     </h2>
-                    <a
-                      className="small link underline"
-                      alt="Project Link"
-                      href={get(work, 'fields.link')}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {get(work, 'fields.linkLabel')}
-                    </a>
+                    <div className="flex flex-row flex-wrap">
+                      {get(work, 'fields.caseStudySlug', '') && (
+                        <Link
+                          className="small link text-no-decoration pb1 pr2"
+                          ariaLabel="View case study"
+                          to={get(work, 'fields.caseStudySlug', '')}
+                          target="_blank"
+                          rel="noopener noreferrer"
+                        >
+                        → View Case Study
+                        </Link>
+                      )}
+                      <a
+                        className="small link underline"
+                        ariaLabel="Project Link"
+                        href={get(work, 'fields.link')}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        {get(work, 'fields.linkLabel')}
```

Comment:
We should render empty string as fallback or checker so that it doesn't render label as `undefined`